src/lib/canvas/utils/index.js

```diff
@@ -44,7 +44,7 @@ export const updateNodeTabs = (layout, nodeId, tabs) => {
   const clonedLayout = cloneDeep(layout);
   const update = (node) => {
     if (node.id === nodeId) {
-      node.tabs = tabs.map(tab => ({ ...tab, id: guid() }));
+      node.tabs = tabs.map(tab => ({ ...tab }));
```

Comment:
do you need to copy this **tab**?

Reply:
no, I need just rewrite tab in a specified node.
and use just destructor to prevent data mutation.
but actually, I use cloneDeep before...so it's not useful.
I will rename it to ```node.tabs = tabs```